runtimevar/blobvar/blobvar_test.go

```diff
@@ -83,7 +83,7 @@ func (verifyAs) SnapshotCheck(s *runtimevar.Snapshot) error {
 
 func (verifyAs) ErrorCheck(w driver.Watcher, err error) error {
 	var perr *os.PathError
-	if !runtimevar.ErrorAs(err, &perr) {
+	if !runtimevar.New(w).ErrorAs(err, &perr) {
```

Comment:
It would probably be better to change `ErrorCheck` to take the concrete type instead of the driver.

Reply:
Done.